### ISSUE TITLE
"More options" navigation

### ISSUE THREAD

**Issue by shaunan** [StatCan/StatcanPublicationUAT#611]
After selecting the “See more” link on the “For you” Publications view, there is a three horizontal dot icon to activate the “More options” sub-menu, however this item that displays when tapping “More options” can’t be navigated to unless navigating through the entire screen first. It should be the next item in the reading order after selecting the three dots.. In addition, the “X” symbol may resemble a “Close” symbol that may confuse users. 

Ensure that all users can navigate the content sequentially as it is presented. When the “More options” items display, focus should land on these before navigating to the list of publication items.

![GitHub24](https://github.com/StatCan/StatcanPublicationUAT/assets/115822055/4c9fe3ab-6da9-4d85-961e-f8a99808e2df)
